Telegram/BotProject/greeting.rb

```diff
@@ -0,0 +1,9 @@
+class Greeting
+  def hello_message(user_name)
+    HELLO_STR + user_name + HELLO_EMOJI
```

Comment:
Add commands, which bot allows to use
e.g.
/weather - I will ask you about the city and will show actual weather
...
etc